### ISSUE TITLE
Training Questions

### ISSUE THREAD

**Issue by JunHyunjae** [isarandi/metrabs#10]
Hi :)
I am pleased with the results of demo_with_just_an_image() in your demo.py.

Q1. Do I run the command below to achieve the same performance as the metrabs_multiperson_smpl_combined model?

./main.py \
  --train --dataset=muco-17-150k --dataset2d=mpii-yolo --scale-recovery=metrabs \
  --epochs=24 --seed=1 --background-aug-prob=0.7 --occlude-aug-prob=0.3 \ 
  --stride-test=32 --logdir=muco/metrabs_univ_seed1 --universal-skeleton

Q2. Do we only need MuCo and MPII as training data?

Q3. Where is a mpii_process_multiperson_train_set.patch for get_much.sh?

Q4. Code of original Metro uses a variety of training sets. (https://github.com/isarandi/metro-pose3d)
      However, training code of this repo uses only h36m or mpi-inf-3dhp datasets. 
      Is the performance still good?

    ex)  ./main.py  --train --dataset=mpi-inf-3dhp --train-on=trainval --epochs=27 --seed=1  --background-aug-prob=0.7 --universal-skeleton --logdir=3dhp/metro_univ_seed1

**Comment by isarandi** [comment]
Hi! Thanks for your interest in the method!

Q1. Reproducing metrabs_multiperson_smpl_combined is not possible with the current codebase because that model was created with more datasets for training, namely H36M, MPI-INF-3DHP, CMU-Panoptic, SAIL-VOS, SURREAL and COCO. As of yet, I've only released the dataset preprocessing code for H36M and MPI-INF-3DHP out of those since the preprocessing is a tedious multi-step process and the main results of the MeTRAbs paper are only based on H36M and MPI-INF-3DHP.

Q2. Only MuCo and MPII are needed for the experiments from the paper where testing is done on MuPoTS-3D. For the 3DPW Challenge there was no restriction in what training data to use, hence I used the datasets listed above in Q1.

Q3. Hm, somehow I missed it. It's fixed now, thanks for letting me know!

Q4. That repository was created for the FG'20 publication (https://arxiv.org/abs/2003.02953) and generally should not be used anymore, it's more for documentation purposes. The current repository (for the TBIOM publication https://arxiv.org/abs/2007.07227) is intended to supersede it and is able to perform those experiments as well. However, as I said in Q1, the performance that won the 3DPW Challenge is only possible to achieve with additional datasets (again, additional data was allowed there and the second placed participant also used several additional datasets).
